lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -1,3 +1,5 @@
+const cancelLikeTerms = require('../../simplifyExpression/fractionsSearch/cancelLikeTerms.js');
+//const removeUnnecessaryParens = require('../../util/removeUnnecessaryParens.js');
```

Comment:
never mind this, it's just to remove the parentheses after multiplication. but i don't think we need to. i will remove this line :D